contracts/Positions/PositionalMarketManager.sol

```diff
@@ -165,6 +186,41 @@ contract PositionalMarketManager is Initializable, ProxyOwned, ProxyPausable, IP
         return false;
     }
 
+    function _checkStrikePrice(
+        address[] memory markets,
+        uint upperPriceLimit,
+        uint lowerPriceLimit
+    ) internal view returns (bool) {
+        for (uint i = 0; i < markets.length; i++) {
+            if (marketsStrikePrice[markets[i]] <= upperPriceLimit && marketsStrikePrice[markets[i]] >= lowerPriceLimit) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    function _checkMarkets(
```

Comment:
looks good, but please add a variable "maxMaturityLength" that disables creating markets e.g. more than 1 month into the future